gemrb/unhardcoded/shared/colors.2da

```diff
@@ -5,3 +5,4 @@ TOOLTIP			0xffffffff
 TOOLTIPBG		0x00000000
 MAPICNBG		0xffffffff
 MAPTXTBG		0x00000000
+ALTDOOR		    0xff0000ff
```

Comment:
Since it's the shared file, this assume the color was red in all other games, so iwd2 *and* iwd1. I've now checked iwd1 and it matches, so all good. :)